setup.cfg

```diff
@@ -30,7 +28,7 @@ zip_safe = True
 packages = find:
 package_dir =
     =src
-python_requires = >=3.7
+python_requires = >=3.9,<3.12
```

Comment:
Is this included because 3.12 does not work or are we not explicitly building for that release?

Reply:
I'm a very strong believer of saying: If a library is not tested for a version then it does not support a version.